src/internal/tokenizer_test.ts

```
@@ -0,0 +1,2674 @@
+
+    transformer.start();
+
+    transformer.flush({
+      enqueue() {
+        throw new AssertionError(
+          "controller.enqueue() shouldn't have been called",
+        );
+      },
+      error(error: unknown) {
+        throw new AssertionError(
+          "controller.error() shouldn't have been called: " + error,
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)